BushBlitz/ev-training-exercise
# Feature/evobe 107/create employee initial data

### Ticket
https://phunts.atlassian.net/browse/EVOBE-107

### Changes
- Created sample `JSON` file for Employees
- Created model for Employee object

### Steps to Test
1. Pull branch
2. go to `src/app/data`
3. `employees.json` should be present containing some sample data of employees having `id`,`firstName` and `lastName` attributes
4. go to `src/app/model`
5. `employee.model.js` should be present with `id`,`firstName`, and `lastName` attribute

### Resources
None
